prime-angular-frontend/src/app/modules/enrolment/shared/services/enrolment-form-state.service.ts

```diff
@@ -515,6 +524,14 @@ export class EnrolmentFormStateService extends AbstractFormStateService<Enrolmen
     });
   }
 
+  public buildEnrolleeHealthAuthorityFormGroup(): FormGroup {
+    // Needs to have the fields to carry data to back-end
+    return this.fb.group({
+      enrolleeId: [null, []],
```

Comment:
You won't need to include this in the form as it can be added as it's transformed in the `json` method of the `EnrolmentFormStateService`. An example, would be `convertSelfDeclarationsToJson` or `convertRemoteAccessSitesToJson` in the `EnrolmentFormStateService`.

Reply:
It seems I still need `buildEnrolleeHealthAuthorityFormGroup` in `patchForm`